pkgs/development/libraries/cosmopolitan/default.nix

```
@@ -24,11 +29,17 @@ stdenv.mkDerivation rec {
   dontConfigure = true;
   dontFixup = true;
 
+  postPatch = ''
```

```suggestion
  preCheck = ''
```

or is this compiled in at buildTime?